test/index.js

```diff
@@ -20,4 +20,30 @@ describe('analytics', function () {
       assert(a.name == b.name);
     });
   });
+
+  describe('noConflict', function () {
+    beforeEach(function () {
+      analytics = window.analytics;
+    });
+
+    afterEach(function () {
+      window.analytics = analytics;
+    });
+
+    it('should leave window.analytics unchanged', function () {
+      // Overwrite window.analytics with random object
+      var dummy = {'dummy': true};
+      window.analytics = dummy;
+      assert(window.analytics === dummy);
+
+      // Unfortunately loading scripts in doesn't work inside a test
+      window.analytics = require('../lib/index.js');
```

Comment:
Using load script won't work as it loads it async and putting assertions in callback leads to next run loop execution which is outside the scope of mocha so no failures are detected.